consensus/dummy/dynamic_fees.go

```diff
@@ -37,22 +37,37 @@ var (
 
 // CalcExtraPrefix takes the previous header and the timestamp of its child
 // block and calculates the expected extra prefix for the child block.
```

Comment:
should we also add `which maybe nil` comment here as well?

Reply:
We could, I felt like a `nil` `[]byte` was more normal (because a `nil` slice is essentially equivalent to an empty slice.)